web/src/index.css

```diff
@@ -28,7 +28,7 @@ code {
 
 .logo {
   background: url("https://cdn.casbin.org/img/casdoor-logo_1185x256.png");
-  background-size: 130px, 27px;
+  background-size: 130px, 27px !important;
```

Comment:
Why add this?

Reply:
![image](https://user-images.githubusercontent.com/41513919/188300383-233254bb-adb2-45e1-bbca-dc8d64e6531c.png)

set `background` with inline style sheet, the `background` property include `background-size`, 
That is to say, the `background-size` set by the class(`.logo`) will be overwritten by the `background-size` set by the inline style sheet, if no `!important` add.